services/navigation/builtin/builtin.go

```diff
@@ -400,19 +400,18 @@ func (svc *builtIn) startWaypointExperimental(extra map[string]interface{}) erro
 		defer svc.activeBackgroundWorkers.Done()
 
 		navOnce := func(ctx context.Context, wp navigation.Waypoint) error {
-			currentLoc, err := svc.Location(svc.cancelCtx, extra)
-			if err != nil {
-				return err
+			if extra == nil {
+				extra = map[string]interface{}{"motion_profile": "position_only"}
+			} else if extra["motion_profile"] == nil {
```

Comment:
Thinking about this slightly more; I think I put the wrong thing in the example. I think for this specific conditional we want 

`} else if _, ok := extra["motion_profile"] ; !ok {`

i.e. if the user actually populated `"motion_profile"` with anything (or nothing!), then leave it alone, but if the keyword is fully unspecified then fill it in.